src/mmw/js/src/draw/views.js

```diff
@@ -176,7 +169,55 @@ var PlaceMarkerView = Marionette.ItemView.extend({
     }
 });
 
-function clearLayer(layer) {
+function changeOutlineLayer(endpoint,select) {
+    var map = App.getLeafletMap(),
+        ol = App.outlineLayer,
+        grid = App.utfGrid;
```

Comment:
Unclear why we are attaching things to `App`. We should try to keep these layers localized to this module / view.